test-utils/test-runner/src/client.rs

```diff
@@ -105,8 +106,14 @@ where
 			default_config(task_executor, chain_spec),
 	};
 
+	let executor = NativeExecutor::<T::Executor>::new(
```

Comment:
I find the naming a little confusing there, passing WASM data to create a **Native**Executor.